guides/advanced_guides/configuration.md

```diff
@@ -133,7 +133,7 @@ In production mode, the [web interface](/guides/advanced_guides/web_interface.md
 
 The maximum size, in bytes, of accepted JSON payloads.
 
-**Default value**: `10485760` (+=10MB)
+**Default value**: `104857600` (+=100MB)
```

Comment:
I'm unfamiliar with the notation `+=X`. Is it meant to convey "approximately X"? I have always used `~X` for that meaning.

Reply:
My bad, it wanted to write `+-` but wrote `+=` instead. And I use `+-` because in french you literally say `plus ou moins`, which translates to `plus or minus`. I'm not sure that exists in english. Either way `~` seems like a more international way of representing approximation.

Reply:
I totally understood what was meant by +-, but maybe that's because I know the expression. Fun fact: the same expression exists in English ("more or less"), but it doesn't have the connection to mathematics like in French.

I'm honestly not sure which is better 🤔 Neither is technically correct, but I guess you're right that ~ could be more universal, since +- is often used to convey a range in either direction. I'll make an issue to change instances of +- to ~.

Reply:
https://github.com/meilisearch/documentation/issues/683#issue-780849219